src/include/gurt/debug.h

```diff
@@ -302,6 +302,12 @@ do {									\
 #define DF_U64		"%" PRIu64
 #define DF_X64		"%" PRIx64
 
+#define D_DEPRECATED(replacement)						\
```

Comment:
It is probably better to use sizeof(replacement) == 1 here since you are passing constant strings.   That way the compiler can determine the length for you and no need to make a runtime function call and deal with any potential SDL issues.

Reply:
Done.